catalyst/rl/environments/gym_wrapper.py

```diff
@@ -46,3 +46,47 @@ def step(self, action):
         info["reward_origin"] = reward
         reward *= self.reward_scale
         return observation, reward, done, info
+
+
+class GymWrapperDiscrete:
```

Comment:
why should we have new wrapper? looks like only `observation_shape` and `num_actions` differs.
Is it okay to define them based on their classes? or make some general function to get them from any gym env, cause they are quite standard.

Reply:
btw, we also can introduce some kind of env specification like `type = 'discrete' or 'continuous'`

Reply:
addressed in the last commit